pkg/kudoctl/packages/verifier/template/verify_render.go

```diff
@@ -70,3 +60,40 @@ func templateCompilable(pf *packages.Files) verifier.Result {
 
 	return res
 }
+
+func collectPipes(pf *packages.Files) map[string]string {
```

Comment:
Instance controller already has [paramsMap](https://github.com/kudobuilder/kudo/blob/master/pkg/controller/instance/instance_controller.go#L400) and [pipesMap](https://github.com/kudobuilder/kudo/blob/master/pkg/controller/instance/instance_controller.go#L424) methods which already implement current logic of filling both maps. Do you think we should reuse them?

Reply:
Done.